chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/DeveloperManager.java

```diff
@@ -389,11 +389,21 @@ public DeveloperDTO merge(List<Long> developerIdsToMerge, DeveloperDTO developer
     }
 
     /**
+<<<<<<< HEAD
```

Comment:
Conflict?

Reply:
Ugh. Eclipse didn't show it to me, and why would I look for conflicts in documentation that I didn't touch? Sigh